### PR TITLE
Feature/manager

### LINKED ISSUE #7: Workflow
You will want to submit PRs to your accountabilibuddy to:

- You must give your accountabilibuddy collaboration access to your repo.
- You must submit at least 2 PRs to your accountabilibuddy for review and allow THEM to merge the PRs you submit.

It is up to you to decide what changes warrant a PR – remember we want to submit PRs that have significant changes and potential for feedback. Think about what functionality you’re struggling with or have questions about or need help with. As an accountabilibuddy, you are responsible for reviewing at least 2 PRs from your partner.

### PR DESCRIPTION
**Pull request by qrispi**
Files Changed: Manager.js, Manager-test.js, index.html, scripts.js, api-calls.js, test-data.js, Customer.js, Hotel-test.js, Booking-test.js

Change Summary: This merge adds a manager class, test file and basic html frame for the manager data. Api calls was also updated to return all customer data for the manager. The manager class is capable of storing all customers, updating their data, and finding a customer by name. Customer was updated to now have a property of money spent instead of returning that from a method. Test-data, Hotel-test, and Booking-test were all modified to use relevant test data.

Functions and methods:

- updateCustomers: calls customer methods to update their bookings with relevant data and update their properties with the total amount they have spent.
- searchCustomers: finds the customer that matches the provided input name (case insensitive) and returns the full customer object.

Closes #7

### CODE REVIEW

**Comment by qrispi** [comment]
I still need to connect all this logic to the DOM but I think the manager class is solid! Let me know what you think @Zertroz !
@robbiejaeger

File: src/classes/Manager.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            customer.calculateMoneySpent();
+        }); 
+    }
+
+    searchCustomers(name) {
+        return this.customers.find(customer => name.toLowerCase() === customer.name.toLowerCase());
+    }
+}
+
+export default Manager;
```
Comment (Zertroz) [inline src/classes/Manager.js:20]: Like the class, keeping it simple but useful

File: test/Booking-test.js [hunk 2]
```diff
@@ -28,24 +28,24 @@ describe('Booking', () => {
     });
 
     it('should store the user ID of who booked it', () => {
-        expect(booking1.userID).to.equal(9);
+        expect(booking1.userID).to.equal(1);
```
Comment (Zertroz) [inline test/Booking-test.js:31]: The joys of testing...

**Review by Zertroz — APPROVED** [review]
Looks good, nice job reworking tests to adapt to the changes you make to your code.

File: src/classes/Manager.js (hunk 1 above)
Comment (qrispi) [reply src/classes/Manager.js:20]: Thanks Kass!

**Pull request merged**